bogo/src/main.rs

```diff
@@ -611,8 +598,11 @@ fn make_server_cfg(opts: &Options) -> Arc<ServerConfig> {
             server::WebPkiClientVerifier::no_client_auth()
         };
 
-    let cert = load_cert(&opts.cert_file);
-    let key = load_key(&opts.key_file);
+    let cert: Vec<_> = CertificateDer::pem_file_iter(&opts.cert_file)
+        .unwrap()
+        .map(|cert| cert.unwrap())
+        .collect();
```

Comment:
Nit: might just be me, but I like `collect::Vec<_>()` better than `Vec<_> = fdsifhjsifjsdifjds.collect()`.